flow_meter/flow_meter.cpp

```diff
@@ -190,6 +191,11 @@ int parse_plugin_settings(const string &settings, vector<FlowCachePlugin *> &plu
          tmp.push_back(plugin_opt("pstats", pstats, ifc_num++));
 
          plugins.push_back(new PSTATSPlugin(module_options, tmp));
+      } else if (proto == "vpndetector"){
+          vector<plugin_opt> tmp;
+          tmp.push_back(plugin_opt("vpndetector", vpndetector, ifc_num++));
+
+          plugins.push_back(new VPNDetectorPlugin(module_options, tmp));
```

Comment:
```suggestion
      } else if (proto == "vpndetector"){
          vector<plugin_opt> tmp;
          tmp.push_back(plugin_opt("ovpn", ovpndetector, ifc_num++));

          plugins.push_back(new OVPNDetectorPlugin(module_options, tmp));
```

I suggest renaming the plugin because it is related to OpenVPN only. This single patch cannot be applied without other modifications on different places...